### PR TITLE
Release 1.2.1

### PR DESCRIPTION
**Pull request by apeltzer**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/nanostring/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/nanostring _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,20 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.2.1 - 2024-xx-xx - Nanometre
```
Comment (apeltzer) [inline CHANGELOG.md:6]:
```suggestion
## v1.2.1 - 2024-01-18 - Nanometre
```

File: .devcontainer/devcontainer.json [hunk 2]
```diff
@@ -17,11 +18,11 @@
...
                 "python.linting.pydocstylePath": "/opt/conda/bin/pydocstyle",
-                "python.linting.pylintPath": "/opt/conda/bin/pylint"
+                "python.linting.pylintPath": "/opt/conda/bin/pylint",
             },
 
             // Add the IDs of extensions you want installed when the container is created.
-            "extensions": ["ms-python.python", "ms-python.vscode-pylance", "nf-core.nf-core-extensionpack"]
-        }
-    }
+            "extensions": ["ms-python.python", "ms-python.vscode-pylance", "nf-core.nf-core-extensionpack"],
```
Comment (christopher-mohr) [inline .devcontainer/devcontainer.json:25]:
```suggestion
            "extensions": ["ms-python.python", "ms-python.vscode-pylance", "nf-core.nf-core-extensionpack"]
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,20 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.2.1 - 2024-01-18 - Nanometre
+
+### `Added`
+
+- Added the Bioinformatics publication to the citation list
```
Comment (christopher-mohr) [inline CHANGELOG.md:10]:
```suggestion
- [#55](https://github.com/nf-core/nanostring/pull/55) - Added the Bioinformatics publication to the citation list
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,20 @@
...
+- Updated to nf-core template 2.11.1
```
Comment (christopher-mohr) [inline CHANGELOG.md:11]:
```suggestion
- [#55](https://github.com/nf-core/nanostring/pull/55) - Updated to nf-core template 2.11.1
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,20 @@
...
+- Granularized the containers for NACHO and Heatmaps
```
Comment (christopher-mohr) [inline CHANGELOG.md:12]:
```suggestion
- [#58](https://github.com/nf-core/nanostring/pull/58) - Granularized the containers for NACHO and Heatmaps
```

File: lib/WorkflowNanostring.groovy [hunk 6]
```diff
@@ -89,7 +89,6 @@ class WorkflowNanostring {
         // Can use ternary operators to dynamically construct based conditions, e.g. params["run_xyz"] ? "<li>Author (2023) Pub name, Journal, DOI</li>" : "",
         // Uncomment function in methodsDescriptionText to render in MultiQC report
         def reference_text = [
-                "<li>Andrews S, (2010) FastQC, URL: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/).</li>",
                 "<li>Ewels, P., Magnusson, M., Lundin, S., & Käller, M. (2016). MultiQC: summarize analysis results for multiple tools and samples in a single report. Bioinformatics , 32(19), 3047–3048. doi: /10.1093/bioinformatics/btw354</li>"
```
Comment (christopher-mohr) [inline lib/WorkflowNanostring.groovy:92]: Also add the Nacho paper here?

File: modules.json [hunk 7]
```diff
@@ -7,20 +7,18 @@
...
-                    "fastqc": {
-                        "branch": "master",
-                        "git_sha": "bd8092b67b5103bdd52e300f75889442275c3117",
+                        "git_sha": "8ec825f465b9c17f9d83000022995b4f7de6fe93",
                         "installed_by": ["modules"]
                     },
                     "multiqc": {
                         "branch": "master",
-                        "git_sha": "a6e11ac655e744f7ebc724be669dd568ffdc0e80",
+                        "git_sha": "8ec825f465b9c17f9d83000022995b4f7de6fe93",
```
Comment (christopher-mohr) [inline modules.json:15]: I guess this should not be the same checksum

**Review by christopher-mohr — CHANGES_REQUESTED** [review]
Found a couple of minor things

File: .devcontainer/devcontainer.json [hunk 8]
```diff
@@ -17,11 +18,11 @@
                 "python.linting.flake8Path": "/opt/conda/bin/flake8",
                 "python.linting.pycodestylePath": "/opt/conda/bin/pycodestyle",
                 "python.linting.pydocstylePath": "/opt/conda/bin/pydocstyle",
-                "python.linting.pylintPath": "/opt/conda/bin/pylint"
+                "python.linting.pylintPath": "/opt/conda/bin/pylint",
```
[the remark this answers is not in the document]
Comment (apeltzer) [reply .devcontainer/devcontainer.json:21]: Unfortunately not, this was discussed yesterday on Slack :)

File: .devcontainer/devcontainer.json (hunk 2 above)
Comment (apeltzer) [reply .devcontainer/devcontainer.json:25]: Same here :)

**Comment by apeltzer** [comment]
@nf-core-bot pretty please fix linting

**Comment by apeltzer** [comment]
@nf-core-bot fix linting pretty please

**Review by christopher-mohr — APPROVED** [review]

**Review by christopher-mohr — APPROVED** [review]

**Review by maxulysse — APPROVED** [review]

**Pull request merged**